pages/[username]/print.tsx

```diff
@@ -32,7 +32,8 @@ export async function getServerSideProps({
   // Note: add the port to the webURL for local development
   const webURL = `${url.protocol}//${url.hostname}/${username}`
 
-  const qrCodeURL = (webURL + "?lightning=" + lnurl).toUpperCase()
+  // const qrCodeURL = (webURL + "?lightning=" + lnurl).toUpperCase()
```

Comment:
I reached out to the galoy team ([here](https://chat.galoy.io/galoy/pl/x6t9aswimt86uy674emsxko98h)) regarding this link and they mentioned this URL is part of the luds fallback schema: https://github.com/lnurl/luds/blob/0318227b280c972ff71202b71072cc91c1b77e5e/01.md?plain=1#L32

So, on one hand, it seems they had a purpose for doing this. On the other hand, when I tested it, it forces you into the Blink UX flow which then requires a lightning wallet to complete. 

So I agree using a lnurl seems like it would be a much simpler UX, but I'm curious if you're familiar with this flow and have any thoughts

Reply:
I'm not that familiar with the flow, and it seems like they are doing some encoding to the lnurl, so I cannot just call it directly. I'm reverting it back for now until we figure out if we can use the fallback schema.

Seems like they are just using a bech32lnurl encoding on the lnurl